v63005/u_inref/gtm8930.csh

```diff
@@ -0,0 +1,67 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo '# Running GTM-8930 subtest to test $VIEW("JNLPOOL") '
+echo '--------------------------------------------------------------------------------'
+echo ''
+
+$MULTISITE_REPLIC_PREPARE 2
+
+echo "# Create a 1 region DB with gbl_dir mumps.gld and region DEFAULT" # , AREG, and BREG"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate_log.txt
+if ($status) then
+	echo "DB Create Failed, Output Below"
+	cat dbcreate_log.txt
+endif
+echo ''
+
+echo "# Start replication"
+$MSR START INST1 INST2
+echo ''
+
+
+echo '# Run test1^gtm8930.m to test $VIEW("JNLPOOL") before and after opening JNLPOOL'
+$ydb_dist/mumps -run test1^gtm8930
+echo ''
+
+echo "# Remove instance file mapping in GDE"
```

Comment:
I want a test where instance file mapping in GDE is removed but gtm_repl_instance is set to an arbitrary file name. In that case, want to see $view("jnlpool") returns the value in the env var.